eng/build.yml

```diff
@@ -37,11 +37,14 @@ phases:
     workspace:
       clean: all
 
-    variables: 
+    variables:
+      - ${{ if parameters.pgoInstrument }}:
+        - _PgoInstrument: '/p:PgoInstrument=true'
+
       - _AgentOSName: ${{ parameters.agentOs }}
       - _TeamName: Roslyn-Project-System
       - _SignType: test
-      - _BuildArgs: '/p:DotNetSignType=$(_SignType)'
+      - _BuildArgs: '/p:DotNetSignType=$(_SignType) $(_PgoInstrument)'
```

Comment:
You need to define an empty `_PgoInstrument` variable, if it is not set at all, the literal `$(_PgoInstrument)` will be used.

Reply:
Huh. Wow.